content/applications/finance/accounting/payables/supplier_bills/ocr.rst

```diff
@@ -41,12 +45,12 @@ Pricing
 =======
 
 | The *Bill Digitalization* is an *In-App Purchase (IAP)* service which requires prepaid credits to
-  work. Digitalizing one document consumes one credit.
+  work. Digitizing one document consumes one credit.
 | To buy credits, go to :menuselection:`Accounting --> Configuration --> Settings --> Bill
   Digitalization` and click on *Buy credits*, or go to :menuselection:`Settings --> Odoo IAP` and
   click on *View My Services*.
 
 .. important::
    - If you are on Odoo Online (SAAS) and have the Enterprise version, you benefit from free trial
      credits to test the feature.
-   - Click `here <https://iap.odoo.com/privacy#header_4>`_ to know about our *Privacy Policy*.
\ No newline at end of file
+   - Click `here <https://iap.odoo.com/privacy#header_4>`_ to learn more about our *Privacy Policy*.
```

Comment:
```suggestion
.. seealso::
   - Our `Privacy Policy <https://iap.odoo.com/privacy#header_6>`_.
```

Since we are speaking of OCR on this page, we should target the OCR header in the privacy policy ;)
Also, the doc generation doesn't like having two identical labels on the same page, that's why the build is red, you have two links using `here` as label ;) (Well, not necessary if you already change the other link label according to my suggestion).